org.eclipse.jdt.ls.target/org.eclipse.jdt.ls.tp.target

```diff
@@ -33,14 +33,18 @@
             <repository location="http://download.eclipse.org/eclipse/updates/4.10-I-builds/"/>
         </location>
         <location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
-            <unit id="org.eclipse.lsp4j.sdk.feature.group" version="0.5.0.v20180903-1212"/>
-            <repository location="http://download.eclipse.org/lsp4j/updates/releases/0.5.0/"/>
+            <unit id="org.eclipse.lsp4j.sdk.feature.group" version="0.0.0"/>
+            <repository location="https://github.com/kittaakos/lsp4j-type-hierarchy/raw/master/p2/"/>
         </location>
         <location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
-            <unit id="org.eclipse.xtend.sdk.feature.group" version="2.14.0.v20180523-0937"/>
-            <unit id="org.eclipse.xtext.sdk.feature.group" version="2.14.0.v20180523-0937"/>
+            <unit id="org.eclipse.wst.xml_ui.feature.feature.group" version="3.10.0.v201804210200"/>
```

Comment:
should not be necessary anymore, since we now only add the m2e-apt core bundle to the TP https://github.com/eclipse/eclipse.jdt.ls/commit/74a09932d29772d4749e67ac00d3c5968e91e94c